android/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.significo.recco">
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
do we also need this permission in the `android/src/main/AndroidManifest.xml`?

or, on the contrary, should we add a `debug` manifest in the recco ui module as we already have for [main](https://github.com/sf-recco/android-sdk/blob/main/api/ui/src/main/AndroidManifest.xml) so I guess that by adding it we can remove the permission from here, is that correct?